FirebaseAuth/Sources/Backend/FIRAuthBackend.h

```diff
@@ -414,6 +425,16 @@ typedef void (^FIRSignInWithGameCenterResponseCallback)(
  */
 + (void)verifyClient:(FIRVerifyClientRequest *)request
             callback:(FIRVerifyClientResponseCallback)callback;
+
+/** @fn getRecaptchaConfig:callback:
+    @brief Calls the getRecaptchaConfig endpoint, which is responsible for retrieving the recaptcha
+   configs including site key, provider enablement status.
+    @param request The request parameters.
+    @param callback The callback.
+ */
++ (void)getRecaptchaConfig:(FIRGetRecaptchaConfigRequest *)request
```

Comment:
This is in a `#if TARGET_OS_IOS` block. See line 401.

Reply:
Hmm, interesting. I thought there should be something specific to TV_OS. It seems that the CI for iOS and macOS were skipped. Thanks for catching this!